src/screens/Login/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import Auth from '../../helpers/auth';
```

Comment:
Same comment as above. having such core business logic in helpers is not great. Maybe having services or actions (like in Redux) would be better.

Also to avoid the trailing path `../../helpers` , it would be better to add custom aliases